src/pages/main/MainStyles.ts

```diff
@@ -0,0 +1,88 @@
+import { createGlobalStyle } from 'styled-components'
+import reset from 'styled-reset'
+
+const MainStyles = createGlobalStyle` 
+    .relative{position:relative}
+    .absolute{position:absolute}
+    .fixed{position:fixed}
+    
+    .flex-center-between {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+    }
```

Comment:
이렇게 쓰고 싶다면, 관련 `styles/mixin.ts`에 mixin 함수를 만들 수 있을 것 같습니다.

// flex관련 mixin
```suggestion

   interface FlexMixin={
   justifyContent?:'center' | 'flex-end' | 'flex-start' | 'space-evenly'
   alignItems? :'center' | 'flex-end' 
   }
   export const MixinFlex({ alignItems,justifyContent }  : FlexMixin)  {  
   
          return css`
          
         display:flex;
         ${justifyContent && css` justify-content : ${justifyContent}`}
        ${alignItems && css`align-Items : ${alignItems}`}
          `
   
   }
```

이런식으로 확장성있게 개발이 가능합니다!
cc. @eunbae11